vocode/streaming/models/events.py

```diff
@@ -17,6 +17,7 @@ class EventType(str, Enum):
     PHONE_CALL_ENDED = "event_phone_call_ended"
     RECORDING = "event_recording"
     ACTION = "event_action"
+    STATE_AGENT_PUBLISH = "event_state_agent_publish"
```

Comment:
I like that you're creating a new event type, but can rename it to something more descriptive? Maybe `STATE_AGENT_TRANSCRIPT` or `STATE_AGENT_JSON_TRANSCRIPT`